apps/alpha-test-app/src/components/layout-demo/SingleSettingsGroup.vue

```diff
@@ -0,0 +1,44 @@
+<script lang="ts" setup>
+import { OnyxRadioButtonGroup, type SelectionOption } from "sit-onyx";
+import { computed } from "vue";
+import type { Settings } from "./LayoutSettings.vue";
+
+const activeSetting = defineModel<Settings>();
+
+const props = defineProps<{
+  headline: string;
+  options: SelectionOption<undefined>[];
+  horizontal?: boolean;
+}>();
+
+const options = computed<SelectionOption<Settings>[]>(() =>
+  props.options.map(
+    (option) => ({ ...option, value: { [option.id]: true } }) as SelectionOption<Settings>,
+  ),
+);
+
+const settingsToSelection = (settings?: Settings): SelectionOption<Settings> | undefined => {
+  return settings
+    ? options.value.find((option) => option.id === Object.keys(settings)[0])
+    : undefined;
+};
+const selectionToSettings = (selection?: SelectionOption<Settings>): Settings => {
+  return selection && selection.value ? selection.value : {};
```

Comment:
```suggestion
  return selection?.value ?? {};
```

Reply:
Maybe not even worth a dedicated function then^^ We could inline it

Reply:
kept it after the latest change where the "value" was removed (got longer again)